Week-4/Promises.js

```diff
@@ -0,0 +1,25 @@
+function getNumber() {
+  return Math.floor(Math.random() * 10);
+}
+let storedNumber = getNumber();
+let dividedNumber = storedNumber % 5;
+// console.log(getNumber())
+console.log(storedNumber);
+// console.log(dividedNumber)
+let checkNumber = new Promise(function (resolve, reject) {

```

Comment:
can make checkNumber as const because you're not modifying the value. Also checkNumber sounds more like a verb but variable names should be nouns. A function name should be a verb.